src/iop/filmic.c

```diff
@@ -1693,7 +1683,7 @@ void gui_init(dt_iop_module_t *self)
   gtk_box_pack_start(GTK_BOX(self->widget), g->grey_point_source, TRUE, TRUE, 0);
   dt_bauhaus_slider_set_format(g->grey_point_source, "%.2f %%");
   gtk_widget_set_tooltip_text(g->grey_point_source, _("adjust to match the average luminance of the subject.\n"
-                                                      "except in back-lighting situations, this should be around 18%."));
+                                                      "except in back-lighting situations, this should be around 18 %%."));
```

Comment:
Sorry my bad, in tooltip there is no need to have double or quoted %. I'm about to commit a fix with this patchset. I have also removed spaces around \n.